doc/remotes.md

```diff
@@ -0,0 +1,52 @@
+# Remotes
+
+## Introduction
+
+Remotes (machine with LXD installed) is concept used by LXD through which users doesn't have to always provide hostname or IP and then validating certificate information.
+LXD introduce three types of remotes:
+
+- Built-in
+- System
+- User
+
+### Built-in
+
+Built-in remotes are:
+- local (default)
+- images
+- ubuntu
+- ubuntu-daily
+
+Local, ubuntu, ubuntu-daily are static and they can't be modified by user.
+
+### System
+
+System remotes can be managed only manually. Remote configuration path is: `/etc/lxd/config/config.yml`.
+System remotes certifications should be placed in `/etc/lxd/config/servercerts/` and named after remote name e.g. `foo.crt`
+
+An example config is below:
+```
+remotes:
+  foo:
+    addr: https://10.0.2.4:8443
+    auth_type: tls
+    project: default
+    protocol: lxd
+    public: false
+  bar:
+    addr: https://10.0.2.5:8443
+    auth_type: tls
+    project: default
+    protocol: lxd
+    public: false
+    
+```
+
+### User
+
+User level remotes are managed from CLI (`lxc`) with command:
+`lxc remote [command]`
+
+By default configuration file is kept in `~/.config/lxc/config.yml` or in `LXD_CONF` if defined.
+User have possibility to override system remotes (e.g. by running command `lxc remote rename` or `lxc remote set-url`)
+which result to add configuration to their own config. Certification files are copied also.
```

Comment:
"which results in the remote being copied to their own config, including any associated certificates."